modules/core/04-channel/keeper/upgrade_test.go

```diff
@@ -326,6 +326,65 @@ func (suite *KeeperTestSuite) TestChanUpgradeTry() {
 	}
 }
 
+func (suite *KeeperTestSuite) TestWriteUpgradeTry() {
+	var (
+		path            *ibctesting.Path
+		proposedUpgrade types.Upgrade
+	)
+
+	testCases := []struct {
+		name                 string
+		malleate             func()
+		hasPacketCommitments bool
+	}{
+		{
+			"success with no packet commitments",
+			func() {},
+			false,
+		},
+		{
+			"success with packet commitments",
+			func() {
+				// manually set packet commitment
+				_, err := path.EndpointB.SendPacket(suite.chainB.GetTimeoutHeight(), 0, ibctesting.MockPacketData)
```

Comment:
Same nit as https://github.com/cosmos/ibc-go/pull/3976#discussion_r1245294283

We may as well check the sequence with `suite.Require().Equal(uint64(1), sequence)`, I think it can be good practise to strive for the whole "never discard return args" thing, even tho sometimes you're kind of forced to.

Also, we should use `suite.chainA.GetTimeoutHeight()` if sending the packet on `EndpointB` :)